store/types/store.go

```diff
@@ -38,6 +38,48 @@ type Queryable interface {
 //----------------------------------------
 // MultiStore
 
+// StoreUpgrades defines a series of transformations to apply the multistore db upon load
+type StoreUpgrades struct {
```

Comment:
I'd like to see some unit tests on this types and its methods.

Reply:
Yes, will do. Otherwise, there is only some very high-level tests.